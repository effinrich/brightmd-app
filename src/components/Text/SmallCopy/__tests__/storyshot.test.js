import initStoryshots from '@storybook/addon-storyshots'

initStoryshots({
  storyKindRegex: /^Text\/SmallCopy$/
})
