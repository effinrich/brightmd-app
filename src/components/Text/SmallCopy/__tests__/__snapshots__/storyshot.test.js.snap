// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Text/SmallCopy default 1`] = `
.c0 {
  color: rgb(170,170,170);
  font-size: 14px;
  margin-top: 8px;
  margin-bottom: 0.5px;
  text-align: left;
}

<p
  className="c0"
  color="rgb(170, 170, 170)"
  fontSize={1}
>
  Poutine kitsch scenester shaman biodiesel shoreditch VHS, sustainable XOXO hashtag. Selvage shoreditch mustache venmo crucifix. Gochujang keffiyeh trust fund dreamcatcher glossier taxidermy narwhal fam organic post-ironic iceland sustainable.
</p>
`;

exports[`Storyshots Text/SmallCopy w/align center 1`] = `
.c0 {
  color: rgb(170,170,170);
  font-size: 14px;
  margin-top: 8px;
  margin-bottom: 0.5px;
  text-align: center;
}

<p
  className="c0"
  color="rgb(170, 170, 170)"
  fontSize={1}
>
  Poutine kitsch scenester shaman biodiesel shoreditch VHS, sustainable XOXO hashtag. Selvage shoreditch mustache venmo crucifix. Gochujang keffiyeh trust fund dreamcatcher glossier taxidermy narwhal fam organic post-ironic iceland sustainable.
</p>
`;

exports[`Storyshots Text/SmallCopy w/align right 1`] = `
.c0 {
  color: rgb(170,170,170);
  font-size: 14px;
  margin-top: 8px;
  margin-bottom: 0.5px;
  text-align: right;
}

<p
  className="c0"
  color="rgb(170, 170, 170)"
  fontSize={1}
>
  Poutine kitsch scenester shaman biodiesel shoreditch VHS, sustainable XOXO hashtag. Selvage shoreditch mustache venmo crucifix. Gochujang keffiyeh trust fund dreamcatcher glossier taxidermy narwhal fam organic post-ironic iceland sustainable.
</p>
`;

exports[`Storyshots Text/SmallCopy w/color 1`] = `
.c0 {
  color: #ff9900;
  font-size: 14px;
  margin-top: 8px;
  margin-bottom: 0.5px;
  text-align: left;
}

<p
  className="c0"
  color="#ff9900"
  fontSize={1}
>
  Poutine kitsch scenester shaman biodiesel shoreditch VHS, sustainable XOXO hashtag. Selvage shoreditch mustache venmo crucifix. Gochujang keffiyeh trust fund dreamcatcher glossier taxidermy narwhal fam organic post-ironic iceland sustainable.
</p>
`;
